src/js/pannellum.js

```diff
@@ -1400,23 +1401,41 @@ function render() {
         // Keep a tmp value of yaw for autoRotate comparison later
         tmpyaw = config.yaw;
 
+        // prevent showing background (empty space) on left or right by adapting min/max yaw
+        var canvas = renderer.getCanvas(),
+            canvasWidth = canvas.width / (window.devicePixelRatio || 1),
+            canvasHeight = canvas.height / (window.devicePixelRatio || 1),
```

Comment:
Since you're only using the width and height to calculate the aspect ratio, there's no need for using the device pixel ratio.

Reply:
Good point